fecgov/openFEC
# Modify `load_archived_murs` to properly handle MURs with multiple PDFs

- [X] Add test for nested PDFs

## Summary (required)
There are  9 Archived MURs that contain multiple pdf files.  We need to be able to load all associated PDF files when pass in a MUR number to `manage.py load_archived_murs` program with either -f (from MUR number) or -s (specific MUR number)

- One part of the resolution for https://github.com/fecgov/openFEC/issues/3234

_Include a summary of proposed changes._
Change the RegEx within `load_archived_murs` function in load_legal_docs.py to be able to pick up all parts of PDFs associated with a MUR number. Then in `process_murs` go thru a loop to load them one at a time to ES and S3.  

## How to test the changes locally
- Pull down the branch to local environment
- Point SQLA_CONN to DEV DB
- Set up S3 bucket variables
- if you don't already have it locally,  `./manage.py create_archived_murs_index`
- test loading MURS using -s and -f flags 
- ./manage.py load_archived_murs -s 2583
- run your local API and check http://localhost:5000/v1/legal/search/?mur_no= 2583&type=murs

## Impacted areas of the application
List general components of the application that this PR will affect:
Loading of Archived MURs

## Related PRs
List related PRs against other branches:

https://github.com/fecgov/fec-cms/issues/2185
